heatequation.jl

```diff
@@ -0,0 +1,71 @@
+using OrdinaryDiffEq, ModelingToolkit, DiffEqOperators, DomainSets
+# Method of Manufactured Solutions: exact solution
+u_exact = (x,t) -> exp.(-t) * cos.(x)
+
+@parameters t x y
+@variables u(..)
+Dxx = Differential(x)^2
+Dyy = Differential(y)^2
+Dt = Differential(t)
+t_min= 0.
+t_max = 2.0
+x_min = 0.
+x_max = 2.
+y_min = 0.
+y_max = 2.
+
+#xmax = 50.0
+#ymax = 30.0
+#ncells = 50.0
+#dx = x_max / ncells
+
+
+lecturer(x, y) = x > x_max / 2 - dx && x < x_max / 2 + dx && y > y_max / 10 - dx && y < y_max / 10 + dx
+emission(x, y, emisrate) = lecturer(x, y) ? emisrate : 0
+@register emission(x, y, emisrate)
+emisrate = 10;
```

Comment:
Do you need to define emisrate above where it is used? It seems like how it is now might cause an error if you run the file from beginning to end.